queries/php/injections.scm

```diff
@@ -1,3 +1,10 @@
 (text) @html
 
 (comment) @phpdoc
+
+;; regex
+
+((function_call_expression
+  function: (_) @_preg_func_identifier
+  arguments: (arguments . (argument (_ (string_value) @regex))))
+    (#match? @_preg_func_identifier "^preg_"))
```

Comment:
`#lua-match?` is generally preferred because it is faster. `#match?` uses the slower vim regex which supports more features but is a bit slower

Reply:
Updated ! Thanks